src/game/server.cpp

```diff
@@ -3395,7 +3395,8 @@ namespace server
                 }
                 prev.clear();
             }
-            if(!buf.empty()) setmods(sv_previousmaps, buf.c_str());
+            if(!buf.empty())
```

Comment:
While you're at it, please add spaces in between statements such as `if` and `for` and the corresponding conditions.

We really need `clang-format` to auto check changes.

Reply:
The current style is to not have a space there. I'm not a big fan of mixing styles because the point of a style is the consistency that it gives.

Reply:
> The current style is to not have a space there. I'm not a big fan of mixing styles because the point of a style is the consistency that it gives.

That is the codebase's old style: we aren't fans of mixing stuff either but every new piece of code should be adopting the new style.